<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsMultiLimitSetting</class>
 <widget class="QDialog" name="clsMultiLimitSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>258</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置一个限定</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,1,1">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>频率范围：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="ClickedLabel" name="lblFrequency">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>---------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpTraceA">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>第一参数上下限：</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="ClickedLabel" name="lblItem1">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>---------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpTraceB">
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>第二参数上下限：</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="ClickedLabel" name="lblItem2">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>---------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickedLabel</class>
   <extends>QLabel</extends>
   <header>ClickedLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
